src/attrcache.c

```diff
@@ -234,7 +234,6 @@ int git_attr_cache__get(
 		/* remove existing entry */
 		if (file) {
 			attr_cache_remove(cache, file);
-			git_attr_file__free(file); /* offset incref from lookup */
```

Comment:
Hmm.  Yeah, you're right, this is not correct.  But I think Coverity is correct here and there's a code path that will double free `file`, when `attr_cache_lookup_entry` (in `attr_cache_remove`) fails...  I'll look at this more carefully when I have a few minutes.